pkg/scheduler/scheduler.go

```diff
@@ -368,6 +370,36 @@ var defaultQueueingHintFn = func(_ klog.Logger, _ *v1.Pod, _, _ interface{}) (fr
 	return framework.Queue, nil
 }
 
+// TODO: remove this function once all plugins implement the QueueingHintFn for podUpdate event
+// and SchedulerQueueingHints feature gate hits GA
+// defaultQueueingHintFnPodUpdate is the default QueueingHintFn for pod related event.
+// It returns queue on podAdd/Delete events. And do a simple check for podUpdate event.
```

Comment:
```suggestion
// defaultQueueingHintFnPodUpdate is a QueueingHintFn for pod related event 
// which is used when the feature gate for QueueingHint is disabled.
// For Add or Delete events, it always returns Queue.
// For Update events, it filters out some update events for the efficient requeueing. 
// That doesn't break any behaviour before QueueingHint because actually that has been done since before QueueingHint was implemented.
```